spec/index.html

```diff
@@ -10510,30 +10510,49 @@ <h4>Notes</h4>
+        <h3>SPARQL String</h3>
         <p>
-          A <dfn data-lt="SPARQLRequestString">SPARQL Request String</dfn> is
-          a <a>SPARQL Query String</a> or <a>SPARQL Update String</a> and is a Unicode character string
-          (c.f. section 6.1 String concepts of [[CHARMOD]]) in the language defined by the following
-          grammar.</p>
+          <span id="defn_SPARQLRequestString"></span>
+          A <dfn>SPARQL string</dfn> is an
+          <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a> that
+          conforms to the grammar given in this section.
+        </p>
+        <p class="note">
+          An <a data-cite="RDF12-CONCEPTS#dfn-rdf-string">RDF string</a> is
+          a sequence of 
+          <a data-cite="I18N-GLOSSARY#dfn-code-point" class="lint-ignore">Unicode code points</a>
+          which are <a data-cite="I18N-GLOSSARY#dfn-scalar-value" class="lint-ignore">Unicode scalar values</a>.
+          Unicode scalar values do not include the
+          <a data-cite="I18N-GLOSSARY#dfn-surrogate" class="lint-ignore">surrogate code points</a>.
+        </p>
         <p>
-          A <dfn data-lt="SPARQLQueryString">SPARQL Query String</dfn> starts
-          at the <a href="#rQueryUnit">QueryUnit</a> production.</p>
+          <span id="defn_SPARQLQueryString"></span>
+          A <dfn>SPARQL query string</dfn> is a
+          <a>SPARQL string</a> that conforms to the grammar starting at 
+          the <a href="#rQueryUnit">QueryUnit</a> production.
+        </p>
         <p>
-          A <dfn data-lt="SPARQLUpdateString">SPARQL Update String</dfn> starts
-          at the <a href="#rUpdateUnit">UpdateUnit</a> production.</p>
-        <p>For compatibility with future versions of Unicode, the characters in this string may
+          <span id="defn_SPARQLUpdateString"></span>
+          A <dfn>SPARQL update string</dfn> is a 
+          <a>SPARQL string</a> that conforms to the grammar starting at 
+          the <a href="#rUpdateUnit">UpdateUnit</a> production.
+        </p>
+        <p>
+          For compatibility with future versions of Unicode, the characters in this string may
           include Unicode codepoints that are unassigned as of the date of this publication (see
           [[[UAX31]]] [[UAX31]] section 4 Pattern Syntax). For productions with excluded character
           classes (for example <code>[^&lt;&gt;'{}|^`]</code>), the characters are excluded from the
-          range <code>#x0 - #x10FFFF</code>.</p>
+          range <code>#x0 - #x10FFFF</code>.
+        </p>
       </section>
 
       <section id="codepointEscape">
         <h3>Codepoint Escape Sequences</h3>
-        <p>A SPARQL Query String is processed for codepoint escape sequences before parsing by the
+        <p>
+          A <a>SPARQL string</a> is processed for codepoint escape sequences before parsing by the
           grammar defined in EBNF below. The codepoint escape sequences for a SPARQL query string
```

Comment:
What's the difference between `a SPARQL string` and `a SPARQL query string`?

Reply:
A SPARQL string is any input to this section.
The parser has two entry points - query and update. It's two languages that happen to share grammar rules.

A SPARQL query string is specific to the query entry point `QueryUnit`.
A SPARQL update string is specific to the update entry point `UpdateUnit`.

Reply:
According to these documents, the names of these languages are `SPARQL Query` and `SPARQL Update`, not `SPARQL query` nor `SPARQL update`. These capitalizations should be used throughout.

Reply:
Your question was "What's the difference between a SPARQL string and a SPARQL query string?" 
I have answered your question.  Please acknowledge that.

The defined terminology "SPARQL query string" is highlighted _as a unit_.
Any other non linked text is, maybe, juxtaposition or just helpful.

You have requested small PRs.
This PR makes focused changes.
The improvement is to remove "request" which is wrong - "request" is a protocol matter.

Scope-creep is time consuming.

We are running out of time after going in circles for _months_.
This WG can achieve "make better" but not "make perfect".

Reply:
Yes, you have clarified that a "SPARQL strsing" is a superclass, with subclasses of "SPARQL Query string" and "SPARQL Update string".

I maintain that the capitalization I have hsed here is correct, and is what should be used throughout.

If you disagree with this, perhaps an issue should be opened for this purely editorial question, which can be answered during CR.